staging/src/k8s.io/apiserver/pkg/endpoints/handlers/get.go

```diff
@@ -45,6 +45,7 @@ import (
 	utilfeature "k8s.io/apiserver/pkg/util/feature"
 	"k8s.io/component-base/tracing"
 	"k8s.io/klog/v2"
+	"k8s.io/utils/pointer"
```

Comment:
I like the high-level approach - I will let you to clean that up to see where we will end up exactly.